### ISSUE TITLE
clear explanation of tmax statistics

### ISSUE THREAD

**Issue by aschetti** [dmgroppe/Mass_Univariate_ERP_Toolbox#7]
Good day David,
I hope the following question is appropriate for this forum. My colleagues and I have recently submitted a manuscript that includes results from the *t<sub>max</sub>* procedure implemented in your toolbox. One reviewer asked to briefly clarify the procedure. I have drafted a response (tailored to our specific data), based on Blair & Karniski (1993) and one of your papers (Psychophysiology 48, 1711–1725 (2011)). I was hoping you could quickly check it out to see if I made some obvious mistakes or misinterpreted the procedure at any point.
Thanks in advance for any help you can give me!

> The *t<sub>max</sub>* statistic was developed by Blair & Karniski (1993) as a powerful method to analyze ERP waveform difference potentials while simultaneously providing strong control of family-wise error rate. A complete description of this approach is outside of the scope of the present manuscript; for details, we recommend the original paper by Blair & Karniski (1993) and Groppe et al. (2011a, 2011b).
>In brief, we used this test as implemented in the Mass Univariate ERP toolbox:
>1.	compute the grand average across all trials, conditions, and participants (separately for each electrode and time point). With respect to the EPN, we averaged across all negative and neutral conditions (irrespective of size and contrast) and computed their mean difference (for the rationale, see Section 4.5);
>2.	for each time point, compute a *t*-test between the aforementioned average and a test value of zero (i.e., corresponding to no difference with baseline). The resulting *t*-value is stored and named *t<sub>observed</sub>*;
>3.	randomly permute condition labels (i.e., each observation is either assigned its actual value or zero), calculate the *t*-test, and store its corresponding *t*-value;
>4.	repeat *step 3* 5,000 times to create a distribution of the possible *t*-values for these data under the null hypothesis;
>5.	the relative location of *t<sub>observed</sub>* in this empirically generated null distribution provides the *p*-value for the observed data, i.e., how probable the actual difference wave at this specific time point would be if the null hypothesis were true;
>6.	at each time point, repeat this procedure for each electrode and retain only the highest *t*-value (i.e., *t<sub>max</sub>*). The *p*-values for the original observations are derived from the *t<sub>max</sub>* scores.

**Comment by dmgroppe** [comment]
Hi Antonio,
  I'm going to be away from email for a bit due to family obligations. I'll
try to respond as soon as I can though (within 14 days at the latest).
     my apologies for the delay,
          -David

On Tue, Jul 10, 2018 at 4:22 AM, Antonio Schettino <notifications@github.com

**Comment by aschetti** [comment]
Hi David,
no problem, thanks for letting me know.

All the best,
Antonio

**Comment by dmgroppe** [comment]
Hi Antonio,
  My apologies for the delay. Your description is slightly inaccurate. Are
you applying the tmax test to just a single electrode or to multiple
electrodes? If a single electrode, the description below would be accurate.
My edits are in bold:


   1. compute the grand average across all trials, conditions, and
   participants (separately for each electrode and time point). With respect
   to the EPN, we averaged across all negative and neutral conditions
   (irrespective of size and contrast) and computed their mean difference (for
   the rationale, see Section 4.5);
   2. for each time point, compute a *t*-test between the aforementioned
   average and a test value of zero (i.e., corresponding to no difference with
   baseline). The resulting *t*-value is stored and named *tobserved*;
   3. randomly permute condition labels (i.e., each observation is either
   assigned its actual value or zero), calculate the *t*-test *AT ALL TIME
   POINTS*, and store *THE MOST EXTREME T-SCORE ACROSS ALL TESTS,*  *tmax* ;
   4. repeat *step 3* 5,000 times to create a distribution of the possible
   *t*-values for these data under the null hypothesis;
   5. the relative location of *tobserved* in this empirically generated
   null distribution provides the *p*-value for the observed data, i.e.,
   how probable the actual difference wave at this specific time point would
   be if the null hypothesis were true; *DERIVING THE p-VALUES IN THIS WAY
   CORRECTS FOR MULTIPLE COMPARISONS WITH THE SAME DEGREE OF CERTAINTY AS
   BONFERRONI CORRECTION (BUT WITH GREATER STATISTICAL POWER).*

I hope this can still be useful.
   -David

**Comment by aschetti** [comment]
Hi David,
thanks for taking the time to reply. I closed the issue a day ago because the reviewing process has been completed: the paper was published today (https://www.nature.com/articles/s41598-018-30701-5). I hope these inaccuracies will not steer the readers away from the reasons why we chose to identify the ERP components of interest using the mass univariate approach. Also, surely the readers will not just rely on our explanation but follow the references in the bibliography (including, obviously, your papers).

Greets,
Antonio
